docs/support.rst

```diff
@@ -7,9 +7,19 @@ Get support
 Get online support
 ==================
 
-(jitsi online meeting)
+There is a monthly online meeting, which you can join without registration.
+It takes place on every second Monday of a month at HH:MM CE(S)T. You can
 
 User forum
 ==========
 
-(openmod)
+If you have a question and you cannot attend an online meeting you can also
+post your question to the `openmod forum <https://forum.openmod.org/>`__. Make
+sure to use the :code:`oemof` tag for your submission.
```

Comment:
As this is the solph documentation, we should advice to use `oemof-solph` for the tag.